packages/sanity/src/structure/documentActions/PublishAction.tsx

```diff
@@ -153,11 +154,22 @@ export const PublishAction: DocumentActionComponent = (props) => {
       return null
     }
     if (liveEdit) {
+      // Live edit documents are not publishable by this action, they are published automatically
+      return null
+    }
+
+    /**
+     * If there is null compareValue, it means the draft is yet to be saved
+     * and so value has the published ID.
+     * This conditional case is for when the published document is being shown,
+     * and so it should only be true if the draft has been saved (compareValue !== null)
+     */
+    if (isPublishedId(value._id) && compareValue !== null) {
       return {
         tone: 'default',
         icon: PublishIcon,
-        label: t('action.publish.live-edit.label'),
-        title: t('action.publish.live-edit.tooltip'),
+        label: t('action.publish.label'),
+        title: getDisabledReason('ALREADY_PUBLISHED', (published || {})._updatedAt, t),
```

Comment:
very much a nit, but saves us an object allocation
```suggestion
        title: getDisabledReason('ALREADY_PUBLISHED', published?._updatedAt, t),
```